contracts/test_api/test_privileged.cpp

```diff
@@ -0,0 +1,21 @@
+/**
+ * @file
+ * @copyright defined in eos/LICENSE.txt
+ */
+#include <eosiolib/eosio.hpp>
+#include <eosiolib/privileged.h>
+#include "test_api.hpp"
+
+/*
+void test_privileged::set_privileged() {
+   prints("hello\n");
+}
+*/
+
+void test_privileged::test_is_privileged() {
+   prints("HELLO\n");
```

Comment:
we should probably have these tests run silently instead of printing out debugging totems.

Reply:
These tests are not finished yet, and I didn't mean to include these with this PR.